src/arrayMethodSort.js

```diff
@@ -4,8 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    const comparison =
+      compareFunction || ((a, b) => (a.toString() > b.toString() ? 1 : -1));
```

Comment:
The `comparison` variable is not being assigned correctly. You need to assign the `compareFunction` or the default comparison function to `comparison`. The current syntax will result in an error.